File: 5-Matematica/ex1761.c
#include <stdio.h>
#include <math.h>

#define PI 3.141592654

int main(){
    //variaveis
    float angulo, distanciaArvore, alturaElfo;
    float alturaArvore, qntdCordao;

    //entrada
    while (scanf("%f %f %f", &angulo, &distanciaArvore, &alturaElfo) != EOF){
        //processamento
        alturaArvore = tan(angulo * (PI / 180.0)) * distanciaArvore + alturaElfo;
        qntdCordao = alturaArvore * 5.0;

        //saida
        printf("%.2f\n", qntdCordao);
    }
    
    return 0;
}

File: 5-Matematica/ex1761.py
from math import tan, radians
#entrada
while True:
   try:
      angulo, distanciaArvore, alturaElfo = str(input()).split()
      angulo = float(angulo)
      distanciaArvore = float(distanciaArvore)
      alturaElfo = float(alturaElfo)

      #processamento
      alturaArvore = tan(radians(angulo)) * distanciaArvore + alturaElfo
      qntdCordao = alturaArvore * 5

      #saida
      print('{:.2f}'.format(qntdCordao))
   except EOFError:
      break
